catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -254,6 +258,9 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
           totalHits = docs.getNumFound();
           addDocsToResults(docs, results);
         }
+
+        List<String> solrQuery = parseSpellcheckedQuery(query.get("q"));
+        responseProps.put("query", (Serializable) solrQuery);
```

Comment:
This is probably a better option. I can do this if you think it's better to put the `Showing Results for` login from `result-item.collection.tsx` into the backend here, and then send the exact String to the front end, so that the front end only has to worry about rendering the String itself.